src/materials/getting-started/javascript.html

```diff
@@ -0,0 +1,4 @@
+---
+notes: |
+  Using Javascript
```

Comment:
@fuhton Are we adding documentation for the ripple documentation in this template?